README.md

```diff
@@ -33,10 +33,10 @@ There are a few requirements depending on your target framework version.
     </PropertyGroup>
     ```
 
-- .NET framework `4.8`
+- .NET framework `4.6.1`
     ```xml
     <PropertyGroup>
-        <TargetFramework>net48</TargetFramework>
+        <TargetFramework>net461</TargetFramework>
```

Comment:
Was there a requirement to support `net461`? I think it already reached [end of life](https://en.wikipedia.org/wiki/.NET_Framework_version_history). Also as far as the `.NET Framework` is concerned all up-to-date Windows machines should be running the latest version anyway 🤷 . So `net48` is fine. 
FIY `.NET Framework` is coupled with Windows and its update cycle, `dotnet` on the other hand is cross-platform and is decoupled from windows.

Reply:
It was a requirement from either NordVPN or NordLayer app, I can't remember exactly but maybe NordLayer?

Reply:
This should be merged, I kinda forgot to merge it because generated code was already net461 compliant :sweat_smile: Actually it might have been NordVPN app installer that requires net461.